src/tests/multihost/alltests/test_config_merging.py

```diff
@@ -87,6 +89,7 @@ def test_0003_lastreadparameter(self, multihost):
             multihost.client[0].put_file_contents(snippet_file, content)
             chmod = 'chmod 600 %s' % snippet_file
             multihost.client[0].run_command(chmod)
+        multihost.client[0].run_command(f"chmod o-rwx {snippet_file}")
```

Comment:
This way you chmod() only last file.
Instead modify `chmod` variable 2 lines above.